test/e2e/pages/allocation-criteria.js

```diff
@@ -72,6 +87,57 @@ class AllocationCriteriaPage extends Page {
       }
     }
 
+    switch (estateAnswer) {
+      case 'Adult male': {
+        fieldsToFill.prisonerCategory = {
+          selector: this.fields.prisonerCategoryAdultMale,
+          type: 'radio',
+        }
+        break
+      }
+
+      case 'Adult female': {
+        fieldsToFill.prisonerCategory = {
+          selector: this.fields.prisonerCategoryAdultFemale,
+          type: 'radio',
+        }
+        break
+      }
+
+      case 'Youth male': {
+        fieldsToFill.prisonerCategory = {
+          selector: this.fields.prisonerCategoryYouthMale,
+          type: 'radio',
+        }
+        break
+      }
+
+      case 'Youth female': {
+        fieldsToFill.prisonerCategory = {
+          selector: this.fields.prisonerCategoryYouthFemale,
+          type: 'radio',
+        }
+        break
+      }
+
+      case 'Other': {
+        fieldsToFill.otherEstate = {
+          selector: this.fields.estate.find('#estate-other'),
+          value: faker.lorem.sentence(6),
+        }
+        break
+      }
+
+      case 'Juvenile male':
+        break
+
+      case 'Juvenile female':
+        break
+
+      default:
+        break
+    }
```

Comment:
Wow. Our helpers...  aren't... helping.

That aside, could this not be achieved with an object literal?

```js
const prisonerCategory = {
  'Adult male': 'prisonerCategoryAdultMale',
  'Adult female': 'prisonerCategoryAdultFemale',
  'Youth male': 'prisonerCategoryYouthMale',
  'Youth female': 
  }
}

const otherEstate = {
  'Other': 
}

if (prisonerCategory[estateAnswer]) {
  fieldsToFill.prisonerCategory = {
    selector: this.fields[prisonerCategory[estateAnswer]],
    type: 'radio',
  }
}

if (prisonerCategory[estateAnswer]) {
  fieldsToFill.otherEstate = {
    selector: this.fields.estate.find('#estate-other'),
    value: faker.lorem.sentence(6),
  }
}
```

Reply:
I refactored it a bit but left the switch.
I have a preference for the switch, but I don't mind making the change since they are the same thing.